integreat_cms/static/src/js/forms/tinymce-init.ts

```diff
@@ -58,10 +58,25 @@ const toggleNoTranslate = (editor: Editor) => {
 
 export const getContent = (): string => tinymce.activeEditor.getContent();
 
+const monkeypatchEditor = () => {
```

Comment:
Could you please clarify for which case we need this monkeypatch ?
It seems to me that changing the listener of the contentChanged event does the trick. Or am I missing something?

Reply:
True, if I didn't miss anything just now, it should suffice for #2300 specifically.
I made the monkey-patch for `isDirty()` because this seems to be where the functionality should reside, but I agree that doing a monkey-patch is also not really a clean approach to do this. But it looks to me like the #2132 (or the underlying mechanism) should/could make use of this functionality?

Reply:
I opened a new PR #2435 because #2132 has stucked due to Firefox. If the new PR looks good then you don't have to think about influence on `dirty`.

Reply:
This was removed and put (further polished) in #2476